docs/extensions.md

```
@@ -0,0 +1,192 @@
+java_library(
+    name = "lib",
+    srcs = glob(["src/main/java/**/*.java"]),
+    deps = ["//greeting-extension/runtime", ...],
+)
+
+quarkus_app(name = "app", deps = [":lib"])
+```
+
+On startup the extension's feature appears in the banner:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)